src/Stripe.net/Services/Plans/StripePlanListOptions.cs

```diff
@@ -4,5 +4,10 @@ namespace Stripe
 {
     public class StripePlanListOptions : StripeListOptions
     {
+        /// <summary>
+        /// Only return plans for the given product.
+        /// </summary>
+        [JsonProperty("product")]
+        public string ProductId { get; set; }
```

Comment:
we should add `created` while we're at it like we do for charges: https://github.com/stripe/stripe-dotnet/blob/master/src/Stripe.net/Services/Charges/StripeChargeListOptions.cs

Reply:
👍  Nice catch!